regulations/templatetags/citation.py

```diff
@@ -0,0 +1,29 @@
+import re
+from django import template
+from django.template.defaultfilters import stringfilter
+from django.utils.safestring import mark_safe
+from django.utils.html import conditional_escape
+
+register = template.Library()
+
+el = ('<a href="https://federalregister.gov/citation/', '">', '</a>')
+
+
+@register.filter(name='citation', needs_autoescape=True, is_safe=True)
+@stringfilter
+def citation(citation, autoescape=True):
+    if autoescape:
+        citation = conditional_escape(citation)
+    citation = linkify(citation)
+
+    return mark_safe(citation)
+
+
+def linkify(citation):
```

Comment:
looks like [inclusion tags](https://docs.djangoproject.com/en/3.2/howto/custom-template-tags/#inclusion-tags) could be another way to do this.